independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/CreateProject.java

```diff
@@ -73,6 +75,22 @@ public CreateProject version(String version) {
         return this;
     }
 
+    public CreateProject name(String name) {
+        if (name == null || name.isBlank()) {
+            return this;
+        }
+        setValue(PROJECT_NAME, name);
```

Comment:
@gsmet `setValue` is already checking for null, maybe we should also check for blank?

Also maybe renaming it to `setValueIfNotBlank`